.github/CODEOWNERS

```diff
@@ -1188,9 +1188,11 @@ packages/kbn-monaco/src/esql @elastic/kibana-esql
 ### Observability Plugins
 
 # Observability AI Assistant
-x-pack/test/observability_ai_assistant_api_integration @elastic/obs-ai-assistant
-x-pack/test/observability_ai_assistant_functional @elastic/obs-ai-assistant
-x-pack/test_serverless/**/test_suites/observability/ai_assistant @elastic/obs-ai-assistant
+/x-pack/test_serverless/api_integration/test_suites/common/data_usage @elastic/obs-ai-assistant
```

Comment:
What tests are these? It doesn't look like related to the Obs AI Assistant

Reply:
https://github.com/elastic/kibana/blob/5d77a1a67c5eb243b14ea0613e411cc13ec73406/x-pack/plugins/data_usage/kibana.jsonc#L5

Reply:
Maybe [this](https://github.com/elastic/kibana/pull/200147#discussion_r1846633044) helps @emma-raffenne ?